scout/server/blueprints/managed_variants/controllers.py

```diff
@@ -141,6 +148,32 @@ def upload_managed_variants(store, lines, institutes, current_user_id):
     return new_managed_variants, total_variant_lines
 
 
+def validate_managed_variant(managed_variant_info):
```

Comment:
This could also be easily done in the HTML code by adding the "required" key in the input fields. At that level you could even specify that position should be a number. This way the form won't be submitted until you add all the required fields. So you can skip the form validation after, and also adding the tests to the form validation..

Reply:
https://www.w3schools.com/tags/att_input_required.asp

Reply:
Im sorry, I dont quite follow? How would you do that for the file parsing? We could (and if we dont should) do that for the manual single variant add form, but the file parser would still need a validator?

Reply:
Ah, ok, looks like I skipped making them required in the other form in favour of easier inheritance. I can fix that. It wasn't really in the scope of the PR, but I agree. It is nicer if the user can't even click submit, though there should of course be validation later in order not to trust this to the browser.

Reply:
Ah yes, your are right, I didn't even test the submitted file with missing fields (topic of THIS PR), I've just try to click submit when the field is empty 😅